src/WarriorRogueMage/CharacterCreation/PickTalentsState.cs

```diff
@@ -84,7 +85,7 @@ private void ViewTalentDescription(string talent)
         {
             string talentToView = talent.Replace("view ", string.Empty);
 
-            Talent foundTalent = this.talents.Find(s => s.Name.Equals(talentToView, StringComparison.CurrentCultureIgnoreCase));
+            Talent foundTalent = this.talents.Find(s => s.Name.Contains(talentToView, StringComparison.CurrentCultureIgnoreCase));
```

Comment:
Should we StartsWith OR Contains to prioritize talents starting with it?

Reply:
Like this?
`Talent foundTalent = this.talents.Find(s => s.Name.StartsWith(talentToView, StringComparison.CurrentCultureIgnoreCase) || s.Name.Contains(talentToView, StringComparison.CurrentCultureIgnoreCase));`

Reply:
Yep. The total time spent processing this is inconsequential. The order here means if user just gives like two characters it should result in finding the one they probably meant over one with those teo chars in the middle.

Reply:
OK, I'll make the update. Thanks!